package/usb-bridge/scripts/S41-start-bridge

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+bridgeName="br0"
+etherInterface="eth0"
+
+if [ "$1" == "start" ]; then
+  # Loading g_ether and creating the appropriate network interface
+  modprobe dwc3-of-simple
+  modprobe dwc3-rockchip-inno
+
+  modprobe g_ether dev_addr=12:34:56:78:9a:bc host_addr=12:34:56:78:9a:bd
+  ip address add 192.0.2.1/30 dev usb0
```

Comment:
get rid of static mac and addresses. Your router will allocate IP addresses. You see, if you have more then one device on the network they can't have the same MACs and IP.